lib/configure.js

```diff
@@ -66,6 +68,13 @@ travis.users.sync.post = promisify(travis.users.sync.post);
 // when creating GitHub tokens.
 var noteUrl = 'https://github.com/mozilla/oghliner';
 
+// The message currently displayed with a spinner, if any.  Most async
+// operations don't need to set this, but occasionally one displays a message
+// and then calls a function that displays another message before
+// the operation is complete.  In that case, the function needs to stop
+// the spinner for the first message before it displays the second message.
+var spinnerMessage;
```

Comment:
Could you create a function that does this automatically? Maybe by wrapping cli.spinner.